scripts/conda/k2/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (c)  2021  Xiaomi Corp.       (author: Fangjun Kuang)
+
+set -ex
+
+CONDA_ROOT=$(conda config --show root_prefix | cut -f2 -d' ')
+CONDA_ENV_DIR=$CONDA_ROOT/envs/k2
+
+rm -rf build
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
```

Comment:
I think RelWithDebInfo is probably better than Release for supporting users. It is the same optimization level, but it includes debug symbols.

Reply:
From https://stackoverflow.com/questions/48754619/what-are-cmake-build-type-debug-release-relwithdebinfo-and-minsizerel